src/main/java/com/cloudbees/jenkins/GitHubRepositoryNameContributor.java

```diff
@@ -18,6 +19,8 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.logging.Level;
+import org.slf4j.Logger;
```

Comment:
you are doing something very wrong if you have both slf4j and java util logging in the same class....